frontend/src/metabase/qb/components/TimeseriesFilterWidget.jsx

```diff
@@ -125,6 +125,7 @@ export default class TimeseriesFilterWidget extends Component {
                 triggerClasses={cx(className, "my2")}
                 ref={ref => this._popover = ref}
                 sizeToFit
+                className="DatePopover"
```

Comment:
Yeah I think we talked earlier that as a general principle, passing class names to our React components is considered an anti-pattern. So something like `noMaxWidth` or `maxWidth={null}`would probably be better – maybe the former because then we can have something like `noMaxWidth={isDatePicker}`in case of `FilterWidget`.